src/lib/forms/widgets/custom_fields/CustomFields.js

```diff
@@ -27,16 +27,18 @@ export class CustomFields extends Component {
     try {
       const { sectionsConfig, discoverFieldsConfig } =
         await this.loadCustomFieldsWidgets();
-      const sections = sectionsConfig.map((sectionCfg) => {
-        const paths = includesPaths(sectionCfg.fields, fieldPathPrefix);
-        return { ...sectionCfg, paths };
-      });
+      const sections = await Promise.all(
+        sectionsConfig.map(async (sectionCfg) => {
+          const paths = includesPaths(sectionCfg.fields, fieldPathPrefix);
+          const widget = await this.loadSectionWidget(sectionCfg);
```

Comment:
```suggestion
          const widget = sectionCfg['ui_widget'] ? await this.loadSectionWidget(sectionCfg) : undefined;
```
just to avoid unnecessary loading attempts?

Reply:
it is done inside the loadSectionWidget, no import attemps will go